### ISSUE TITLE
MSI GS75 - Exact same issue

### ISSUE THREAD

**Issue by BasBuur** [dhedlund/kernel-patch-lg-gram-17#1]
Thanks for your work on this issue. I've recently received my new MSI GS75, which seems to have the exact same issue as the LG Gram. Using the `pci=nommconf` parameter also solves the issue for me, but I heavily rely on virtualization for my work, which this parameter breaks.

Have you done any more investigation on this, or have heard of another way to solve this? I'm on Ubuntu in case it matters.

**Comment by jayz28** [comment]
> Thanks for your work on this issue. I've recently received my new MSI GS75, which seems to have the exact same issue as the LG Gram. Using the `pci=nommconf` parameter also solves the issue for me, but I heavily rely on virtualization for my work, which this parameter breaks.
> 
> Have you done any more investigation on this, or have heard of another way to solve this? I'm on Ubuntu in case it matters.

@Moosieboy in case you haven't been able to solve this yet, I have the same issue where I need to be able to run virtualization on my LG Gram 17.  I was able to recompile the Ubuntu 19.04 kernel (5.0.0-20) with the offending commit reverted (84c8b58) and everything so far runs smoothly.  This is the kernel build guide I used: https://wiki.ubuntu.com/Kernel/BuildYourOwnKernel

**Comment by BasBuur** [comment]
> > Thanks for your work on this issue. I've recently received my new MSI GS75, which seems to have the exact same issue as the LG Gram. Using the `pci=nommconf` parameter also solves the issue for me, but I heavily rely on virtualization for my work, which this parameter breaks.
> > Have you done any more investigation on this, or have heard of another way to solve this? I'm on Ubuntu in case it matters.
> 
> @Moosieboy in case you haven't been able to solve this yet, I have the same issue where I need to be able to run virtualization on my LG Gram 17. I was able to recompile the Ubuntu 19.04 kernel (5.0.0-20) with the offending commit reverted (84c8b58) and everything so far runs smoothly. This is the kernel build guide I used: https://wiki.ubuntu.com/Kernel/BuildYourOwnKernel

Yeah, I ended up doing the same, works great. Thanks for the tip!
